src/app.rs

```diff
@@ -321,6 +322,11 @@ fn usage() -> HashMap<&'static str, Help> {
         , "Follow symbolic links"
         , "By default, fd does not descend into symlinked directories. Using this flag, symbolic \
            links are also traversed.");
+    doc!(h, "same-file-system"
+        , "Don't cross file system boundaries (only Unix/Windows)"
+        , "By default, fd will traverse the file system tree as far as other options dictate. \
+           With this flag, fd ensures that it does not descend into a different file system than \
+           the one it started in. Does nothing if not on Unix or Windows.");
```

Comment:
Maybe we could also mention that this is similar to `find`s `-mount` option. That would be useful for users that look for `--mount`. We could even provide `--mount` as a hidden alias.

Reply:
I've decided to provide both `--mount` and `--xdev` as aliases, since `--xdev` is the preferred name, at least according to the man page of GNU find(1).